fb_instant_articles.module

```diff
@@ -8,28 +8,14 @@
  * Implements hook_menu().
  */
 function fb_instant_articles_menu() {
-  // RSS Endpoint.
-  $items['fbinstant.rss'] = array(
-    'page callback' => 'fb_instant_articles_rss_page',
-    'file' => 'includes/rss.inc',
-    'access arguments' => array('access content'),
-  );
-
   // Top level configuration page.
   $items['admin/config/services/fb-instant-articles'] = array(
-    'title' => 'Facebook Instant Articles Settings',
-    'description' => 'Facebook Instant Articles',
-    'page callback' => 'drupal_get_form',
-    'page arguments' => array('fb_instant_articles_admin_settings_form'),
-    'access arguments' => array('administer fb_instant_articles'),
-    'file' => 'includes/admin.inc',
-  );
-
-  // Configure Facebook Instant Article settings.
-  $items['admin/config/services/fb-instant-articles/settings'] = array(
-    'title' => 'Settings',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
-    'weight' => -1,
+    'title' => 'Facebook Instant Articles',
+    'description' => 'Settings related to Facebook Instant Articles.',
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => array('access administration pages'),
+    'file' => 'system.admin.inc',
+    'file path' => drupal_get_path('module', 'system'),
```

Comment:
@m4olivei Well we could do this several ways:

1. Currently in this PR I'm making a menu section placeholder, like the system module does in `system_menu()`, so that other modules (in this case we do this for our own submodules) can add children menu items. Without the children items, there will be no children settings pages. This is what I meant in the initial PR comment – apart from the metatag helper (which i think can be done in a different way somewhere else, but we need to talk about it outside the scope of this PR) I'm not certain yet that the base module needs any settings form at all on it's own. I just left that admin.inc file where it is till we decide how to handle that one variable.
2. Alternately we could create one admin form all on the same page, and then have submodules form alter that to add new sections as needed (adding groupings etc to clarify what each set of configs are for).

I'm not dead set on how we do it, but I did the initial PR this way because it's less tightly coupled (and less unnecessary alters) and in that way I personally find the nicer option of the two. But that's something we can all decide together. But does this help clarify?